### DISCUSSION TITLE
Help with django and jinja2 html file syntax and completions.

### DISCUSSION THREAD

**Discussion by Exoutia** [lcfd/NaryaVim#22] [Q&A]
First sorry I shouldn't have used issue tab to ask for suggestions but I am also trying to make my neovim config for flask and django could you tell me how did you add any support for completions and syntax highlight for django templates and jinja2 html files.

**Comment by lcfd** [comment]
Hi @Exoutia 👋 

Try to look in the LSP configuration.
There should be everything you need to work with Django and the `htmldjango` type.

Also, if you need something different from Narya try to give a look at [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter).
